src/pk_core.c

```diff
@@ -906,6 +906,66 @@ DEF(stdMathTangent,
   RET(VAR_NUM(tan(rad)));
 }
 
+DEF(stdMathSinh,
+  "sinh(rad:num) -> num\n"
+  "Return the hyperbolic sine value of the argument [rad] which is an angle "
+  "expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(sinh(rad)));
+}
+
+DEF(stdMathCosh,
+  "cosh(rad:num) -> num\n"
+  "Return the hyperbolic cosine value of the argument [rad] which is "
+  "an angle expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(cosh(rad)));
+}
+
+DEF(stdMathTanh,
+  "tanh(rad:num) -> num\n"
+  "Return the hyperbolic tangent value of the argument [rad] which is "
+  "an angle expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(tanh(rad)));
+}
+
+DEF(stdMathArcSine,
+  "asin(rad:num) -> num\n"
+  "Return the arcsine value of the argument [rad] which is an angle "
+  "expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(asin(rad)));
+}
+
+DEF(stdMathArcCosine,
+  "acos(rad:num) -> num\n"
+  "Return the arc cosine value of the argument [rad] which is an angle "
+  "expressed in radians.") {
```

Comment:
for `asin acos and atan`the parameters are not angles, but numbers, also we need to state that `the return value is an angle expressed in radians`